packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -302,6 +325,18 @@ class OpPerfTelemetry {
 				opPerfData: {},
 			});
 		}
+
+		if (message.type == MessageType.NoOp) {
+			// Count the number of no-ops submitted by this client.
+			// The value is reset when we log the OpStats sampled event.
+			this.noOpCount++;
+		}
+
+		this.opsLogger.sendPerformanceEvent({
+			eventName: "OpStats",
+			submitOpCount: OpPerfTelemetry.OPS_SAMPLE_RATE,
+			noOpCount: this.noOpCount,
```

Comment:
noOpCount will always be zero, no? Because it's reset before returning true in submitOpEventSampler